src/config.rs

```diff
@@ -3231,6 +3231,21 @@ impl ConfigController {
 
     pub fn update(&self, change: HashMap<String, String>) -> CfgResult<()> {
         let diff = to_config_change(change.clone())?;
+        self.update_impl(diff, Some(change))
+    }
+
+    pub fn update_from_toml_file(&self) -> CfgResult<()> {
+        let current = self.get_current();
+        let incoming = TiKvConfig::from_file(Path::new(&current.cfg_path), None);
```

Comment:
`TiKvConfig::from_file` will panic if there is no such file, better to change it to return an `Err` and we print a log here instead.

Reply:
Fixed, thanks!